### ISSUE TITLE
Oops, something went wrong! (admin)

### ISSUE THREAD

**Issue by MiFrance** [getformwork/formwork#47]
Hi,
I don't have acces to admin, look my log:
PHP 7.3.8
```
[21-Dec-2019 12:04:23 UTC] Uncaught TypeError: Argument 1 passed to Formwork\Admin\Users\Avatar::__construct() must be of the type string, null given, called in /Applications/MAMP/htdocs/formwork/admin/src/Users/User.php on line 50 in /Applications/MAMP/htdocs/formwork/admin/src/Users/Avatar.php:14
Stack trace:
#0 /Applications/MAMP/htdocs/formwork/admin/src/Users/User.php(50): Formwork\Admin\Users\Avatar->__construct(NULL)
#1 /Applications/MAMP/htdocs/formwork/admin/src/Users/Users.php(30): Formwork\Admin\Users\User->__construct(Array)
#2 /Applications/MAMP/htdocs/formwork/admin/src/Admin.php(104): Formwork\Admin\Users\Users::load()
#3 /Applications/MAMP/htdocs/formwork/formwork/Router/Router.php(151): Formwork\Admin\Admin->__construct()
#4 /Applications/MAMP/htdocs/formwork/formwork/Core/Formwork.php(214): Formwork\Router\Router->dispatch()
#5 /Applications/MAMP/htdocs/formwork/index.php(31): Formwork\Core\Formwork->run()
#6 {main}
```

**Comment by giuscris** [comment]
Thank you @MiFrance for reporting this bug. I'm solving this right now.

**Comment by giuscris** [comment]
Resolved in Formwork 1.4.4
